core/trino-main/src/main/java/io/trino/server/security/AbstractBearerAuthenticator.java

```diff
@@ -19,8 +19,10 @@
 import javax.ws.rs.container.ContainerRequestContext;
```

Comment:
Please add a test to `io.trino.server.security.oauth2.TestOAuth2WebUiAuthenticationFilter`

Reply:
see https://github.com/trinodb/trino/pull/9524